### PR TITLE
sd-bus/man: document EBUSY error in bus_message_read

### PR DESCRIPTION
**Pull request by oldaccountdeadname**
I had to dig through the source to find the cause of this error, so hopefully the docs introduced in this commit are correct and would be helpful. Thanks for taking a look at the commit, and happy 2022!

Here's the commit message:

```
The EBUSY error can be returned from sd_bus_exit_container(), and, if
that happens, it will be propogated upwards towards bus_message_read. In
terms of documentation, this means that bus_message_read's man page
can't just include the error text for sd_bus_message_read_basic, as
reading basic types exclusively doesn't have the potential for this
error.

sd_bus_message_read_basic's error documentation isn't incorrect when
applied to sd_bus_message_read, it's just incomplete. This patch
replaces the include directive with a manual copy supplemented with
documentation of EBUSY.
```

### CODE REVIEW

**Comment by lincolnauster** [comment]
Ah, looks like I mistyped a tag. Force-pushed an ammended commit, CI might pass now.

File: man/sd_bus_message_read.xml [hunk 1]
```diff
@@ -172,7 +172,40 @@
...
+        </varlistentry>
+
+        <varlistentry>
+          <term><constant>-EBADMSG</constant></term>
+
+          <listitem><para>The message cannot be parsed.</para></listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><constant>-EBUSY</constant></term>
```
Comment (bluca) [inline man/sd_bus_message_read.xml:185]: Why not simply adding this entry to the existing list in man/sd_bus_message_read_basic.xml ?

**Comment by keszybz** [comment]
Instead of copying the text, please include the descriptions of the individual errors:
```xml
    <refsect2 id='errors'>
      <title>Errors</title>

      <para>Returned errors may indicate the following problems:</para>

      <xi:include href="sd_bus_message_read_basic.xml" xpointer="errors-einval" />
      <xi:include href="sd_bus_message_read_basic.xml" xpointer="errors-enxio" />
      <xi:include href="sd_bus_message_read_basic.xml" xpointer="errors-ebadmsg" />
      ...
      <varlistentry>
          <term><constant>-EBUSY</constant></term>
This conversation was marked as resolved by keszybz

          <listitem><para>When reading from a container, this error will be returned if unread elements
          are left in the container.</para></listitem>
     </varlistentry>
...
```

(This means that id's will need to be added the the varlistentries for the individual errors.)

You can look at `common-variables.xml` for inspiration.

**Comment by lincolnauster** [comment]
> Instead of copying the text, please include the descriptions of the individual errors

I believe this is done in 16d34e0e7a. Local rendering checks out, but apparently there's still a CI fail. The 177k-line log file is a bit much for me to parse, any hints? Thanks for the quick feedback and sorry for the slow patching!

**Comment by yuwata** [comment]
Ubuntu CIs are slow and unstable. Please ignore the failures.

**Review by bluca — APPROVED** [review]

**Pull request merged**